spinoso-string/src/lib.rs

```diff
@@ -1249,6 +1249,36 @@ impl String {
         self.buf.reserve(additional);
     }
 
+    /// Reserves capacity for at least `additional` more elements to be inserted
+    /// in the `String`. The collection may reserve more space to avoid
+    /// frequent reallocations.
+    ///
```

Comment:
I noticed this duplicate bit was removed in another place in one of the commits. Did you mean to remove this as well?

```suggestion
```

Reply:
Yup! I was going to remove this as well. Thanks for pointing it out!